src/botPage/view/View.js

```diff
@@ -238,6 +243,43 @@ const applyToolboxPermissions = () => {
         [fn]();
 };
 
+const checkForRequiredBlocks = () => {
+    const displayError = errorMessage => {
+        const error = new Error(errorMessage);
+        globalObserver.emit('Error', error);
+    };
+
+    const blockLabels = { ...config.blockLabels };
+    const missingBlocksTypes = getMissingBlocksTypes();
+    const disabledBlocksTypes = getDisabledMandatoryBlocks().map(block => block.type);
+    const unattachedPairs = getUnattachedMandatoryPairs();
+
+    if (missingBlocksTypes.length) {
+        missingBlocksTypes.forEach(blockType =>
+            displayError(`"${blockLabels[blockType]}" block should be added to the workspace.`)
```

Comment:
`block should be added to the workspace.` must be localized.

Reply:
yes, sure. I forgot to do it waiting for content team. Thanks